src/plus/webviews/graph/protocol.ts

```diff
@@ -246,6 +247,14 @@ export const DidChangeColumnsNotificationType = new IpcNotificationType<DidChang
 	true,
 );
 
+export interface DidChangeWindowFocusParams {
+	focused: boolean;
+}
+export const DidChangeWindowFocusNotificationType= new IpcNotificationType<DidChangeWindowFocusParams>(
```

Comment:
```suggestion
export const DidChangeWindowFocusNotificationType = new IpcNotificationType<DidChangeWindowFocusParams>(
```

Reply:
Done in commit: `Fixed code formatting issues`.